### DISCUSSION TITLE
Data is being wiped out on Render

### DISCUSSION THREAD

**Discussion by alan313313** [FlowiseAI/Flowise#367] [Q&A] [answered]
Hi,

I have an issues with hosting on Render.

The issue is that I hosted my Flowise on Render, and all my work is consistently being wiped.
I found information that Render is an ephemeral service, and redeploys the app often, which means that everything is gone after redeployment.

I wrote to them explaining the situation and this is their (Render's) response.

"To ensure I understand this, is this what you are trying to run on Render: https://github.com/FlowiseAI/Flowise?  I'm not so sure you can do this on Render.  The problem is the database looks to be an SQLite database stored in $HOME/.flowise: https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/DataSource.ts#L13.  SQLite stores data on your app's disk, which, as you said, is ephemeral by default.  If you could somehow configure this, then you could use a persistent disk to store the SQLite data: [https://render.com/docs/disks,](https://render.com/docs/disks) but you would need to be able to configure the path the database is stored on.
 
Please follow this up with the maintainers for FlowiseAI and ask them to make this location configurable with an environment variable, or support a different database type like Postgres."

Could I get some answers on it please?

**Comment by mencelot** [comment]
I am experiencing the same issue. It runs for a while and then its just gone.

**Comment by marius-smg** [comment]
Yeah for me the same.

**Comment by HenryHengZJ** [comment, marked as answer]
hey @alan313313 @mencelot @marius-smg,

In the latest v1.2.13 release, you can specify an env DATABASE_PATH. This will allow you to specify where the data is being stored. And using Render's Disk, you can mount that path and have persistent storage.

Here's the step by step guide - https://docs.flowiseai.com/deployment/render#persistent-disk

Let us know if that resolves the issue
